packages/elements/src/pill/index.ts

```diff
@@ -42,6 +55,11 @@ export class Pill extends ControlElement {
     return VERSION;
   }
 
+  /**
+   * Element's role attribute for accessibility
+   */
+  protected readonly defaultRole: string | null = 'button';
```

Comment:
Why default role is `button`?

Reply:
Pill is a small button style component, I think that we should set role on the pill as `button` by default

Reply:
It is incorrect when the pill has a clear button because it will be the anti-pattern nested interactive element. IMO the role should be button in the toggle mode only/

Reply:
We agree to set `button` as a default role